_posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Adding video backgrounds with jekyll front-matter"
+date: 2016-11-10 13:27:57
+tags:
+- jekyll
+- video
+- liquid
+authors: Kevin Hougasian
+video: /video/liquid
+---
+Most of our web properties run on [jekyll](https://jekyllrb.com), [ebrokersoftware.com](https://ebrokersoftware.com), [theagentsurvivalguide.com](https://theagentsurvivalguide.com), and more slated for the coming year.
+
+The RIMdev blog lets up both document and play with what may, or may not, work.
+
+## What about Video headers?
+
+Glad you asked! We currently check the post front matter and, if `image:` is listed, serve accordingly.
+
+{% raw %}
+```
+{% if page.image %}
+  {% assign postImage = page.image %}
+{% elsif post.image %}
+  {% assign postImage = post.image %}
+{% else %}
+  {% assign postImage = site.default_image %}
+{% endif %}
+```
+{% endraw%}
+
+{: .ui.info.message }
+Why `page` AND `post`? Our main page is `assigned` [as a post](https://gist.github.com/nimbupani/1421828).
+
+### HTML5 video and fallback
+
+Let's load the video in our front matter, and handle fallback and poster image in the post in our include. We're only dealing with `.mp4` and `.webm` here. Video conversion? [Cloud convert](https://cloudconvert.com/mp4-to-webm) does a decent job for now. So now we have:
+
+```
+---
+layout: post
+video: /video/clouds_over_the_mountain
+---
+```
+We could have used a collection and repeated our 3 formats with the same name, but a single entry with now extention makes more sense while listing the path.
```

Comment:
`extention` should be `extension`. Could you clarity what "with now extension" means?